server/views/partials/layout.njk

```diff
@@ -50,6 +50,14 @@
   }) }}
 {% endblock beforeContent %}
 
+{% block content %}
```

Comment:
I think I'd like us to err on the side of duplication here, rather than adding this in.

Although it's essentially the same line of code on each page, I find it easier to consider the HTML structure of the page here (given this is used in non-partial templates) knowing that there's an `<h1>` explicitly declared in this file. 

Having this new block introduces a bit more complexity as to know there's an `<h1>` here you'd have to know to check the `layout.njk` file.

I know this is inconsistent with how we set `title`s but I consider them less of a "main character" in the structure of an HTML page, if that makes any sense at all.

Reply:
Happy to undo that. What do you think about the alternative I suggested in the commit message? I think I asked about this in an early PR, but page headings seem to be either an outlier as the hardcoded text content that gets passed down from the controller, or using data that's already available from other stuff that gets passed down (e.g. course or organisation name)

(I'd be more keen on this than the abstraction to setting the heading in the layout actually!)

> Alternatively, we could skip passing the heading down as a variable and move the currently controller-based code into the view (`coursePresenter.nameAndAlternateName`, `"List of accredited programmes"` and `"Accredited Programmes"`)